christfellowshipchurch/web-app-v2
# All links on Group Single now open up in a new tab CFDP-1359

## Description 
All links on Group Single now open up in a new tab, including the Resources and the Join Meeting button

## How to Test
1. Open your groups page /groups/cfdp-testing-group
2. Click on any of the resources and they should open up in a new tab
3. Click on the Join Meeting button and they should open up in a new tab

## Jira Ticket
[CFDP-1359]

[CFDP-1359]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1359
